astropy/io/misc/tests/test_pickle_helpers.py

```diff
@@ -18,7 +18,7 @@ def test_fnpickling_simple(tmpdir):
     res = fnunpickle(fn, 0)
     assert obj1 == res
 
-    # now try with a file-like object instead of a string
+    # now try with a file-like instead of a string
```

Comment:
here "object" could stay :)

Reply:
In general, as far as I could see the definitions in `glossary.rst` would automatically convert the `file-like` to "file-like object" along with the reference link wherever it appears in the description strings, but not in comments, C code and it seems not in the general docstrings either, so for all of the latter the original should be kept. Will make it more cumbersome for all the instances changed by automatic replace, but when I have time, I can go through the changes in `io.ascii` for my part and add edit suggestions.

Reply:
@dhomeier, https://github.com/astropy/astropy/pull/11118/commits/0e79d6d0437ccef76bbae41e5b72e6e725feea8a is my first pass at correcting this.

Reply:
✅